### DISCUSSION TITLE
XA60

### DISCUSSION THREAD

**Discussion by jberrebi** [CMRR-C2P/MB#342] [Q&A]
Hi,
We are still running R016a under VE11C but we need to upgrade. Siemens offers us to directly upgrade to XA60 and I guess that we can postpone a little the upgrade. In a previous discussion thread (https://github.com/CMRR-C2P/MB/discussions/288) , @eauerbach wrote _"I still wouldn't recommend for anyone with a functional VE11C or VE11E scanner to upgrade to XA30—wait for XA60 or so next year if you can."_
Is this advice still valid? Should we ask Siemens to postpone the upgrade for a few weeks/months in order to get XA60?
Jonathan

**Comment by eauerbach** [comment]
I personally still have a hard time recommending upgrading from VE to XA in general, but XA60 sequences are now available.
